datasets.md

```diff
@@ -0,0 +1,87 @@
+# Dataset Preparation Guide
+
+If you want to use prepared configs to run the Accuracy Checker tool and the Model quantizer, you need to organize folders with validation datasets in a certain way.  Instructions for preparing validation data are described in this document.
+
+## [ImageNet](http://image-net.org)
+
+### How download dataset
+
+To download images from ImageNet, you need to have an account and agree to the Terms of Access. Follow the steps below:
+1. Go to the [ImageNet](http://www.image-net.org/) homepage
+2. If you have an account, click `Login`. Otherwise, click `Signup` in the right upper corner, provide your data, and wait for a confirmation email
+3. Log in after receiving the confirmation email and go to the `Download` tab
+4. Select `Download Original Images`
+5. You will be redirected to the Terms of Access page. If you agree to the Terms, continue by clicking Agree and Sign
+6. Click one of the links in the Download as one tar file section to select it
+7. Unpack archive
+
+To download annotation files, you need to follow the steps below:
+* `val.txt`
+  1. Download arhive <http://dl.caffe.berkeleyvision.org/caffe_ilsvrc12.tar.gz>
+  2. Unpack archive
+* `val15.txt`
+  1. Download annotation file <https://raw.githubusercontent.com/hujie-frank/SENet/master/ILSVRC2017_val.txt>
+  2. Rename `ILSVRC2017_val.txt` to `val15.txt`
+
+### Structure of directory with dataset
+
+This section describes what structure of directory dataset must have to run the Accuracy Checker tool.
+* `<DATASET_DIR>` - root directory
+    * `ILSVRC2012_img_val` - directory containing the ILSVRC 2012 validation images
+    * `val.txt` - annotation file used for ILSVRC 2012
+    * `val15.txt` - annotation file used for ILSVRC 2015
+
+### Datasets in dataset_definitions.yml
+* `imagenet_1000_classes` used for evaluation models trained on ILSVRC 2012 dataset for 1000 classes. (model examples: alexnet, colorization-v2)
```

Comment:
colorization not trained on this dataset, it uses another model for verification its accuracy
```suggestion
* `imagenet_1000_classes` used for evaluation models trained on ILSVRC 2012 dataset for 1000 classes. (model examples: alexnet,  vgg16)
```

Reply:
Done